--- nut-2.2.0/drivers/megatec_usb.c.open	2007-08-16 09:32:03.000000000 +0200
+++ nut-2.2.0/drivers/megatec_usb.c	2007-08-16 09:32:24.000000000 +0200
@@ -209,7 +209,7 @@
 	/* link the matchers */
 	regex_matcher->next = &subdriver_matcher;
 
-	ret = usb->open(&udev, &hiddevice, regex_matcher, NULL, MODE_NOHID);
+	ret = (usb->open)(&udev, &hiddevice, regex_matcher, NULL, MODE_NOHID);
 	if (ret < 0)
 		usb_open_error(port);
 
--- nut-2.2.0/drivers/libhid.c.open	2007-08-16 09:28:05.000000000 +0200
+++ nut-2.2.0/drivers/libhid.c	2007-08-16 09:28:23.000000000 +0200
@@ -589,7 +589,7 @@
 	}
 
 	/* get and parse descriptors (dev, cfg and report) */
-	ReportSize = comm_driver->open(udevp, hd, matcher, ReportDesc, mode);
+	ReportSize = (comm_driver->open)(udevp, hd, matcher, ReportDesc, mode);
 
 	if (ReportSize == -1)
 		return NULL;
