diff -up nut-2.7.1/clients/Makefile.am.fixupslog nut-2.7.1/clients/Makefile.am
--- nut-2.7.1/clients/Makefile.am.fixupslog	2013-07-26 21:41:10.000000000 +0200
+++ nut-2.7.1/clients/Makefile.am	2014-03-06 14:52:09.625634472 +0100
@@ -45,7 +45,7 @@ upsstats_cgi_SOURCES = upsstats.c upscli
  upsimagearg.h cgilib.c cgilib.h
 
 # not LDADD.
-libupsclient_la_SOURCES = upsclient.c upsclient.h
+libupsclient_la_SOURCES = upsclient.c ../common/common.c upsclient.h
 libupsclient_la_LIBADD = ../common/libparseconf.la
 if WITH_SSL
   libupsclient_la_LIBADD += $(LIBSSL_LIBS)
