--- nut-2.2.0/lib/libupsclient-config.in.wrongssl	2007-09-06 11:10:06.000000000 +0200
+++ nut-2.2.0/lib/libupsclient-config.in	2007-09-06 11:13:06.000000000 +0200
@@ -10,7 +10,7 @@
 Version="@PACKAGE_VERSION@"
 prefix=@prefix@
 exec_prefix=@exec_prefix@
-Cflags="-I@includedir@ @SSL_CFLAGS@"
+Cflags="-I@includedir@ @LIBSSL_CFLAGS@"
 
 arch=$(/bin/arch)
 if [ "$arch" = "x86_64" -o "$arch" = "s390x" -o "$arch" = "ppc64" -o "$arch" = "ia64" ] ; then
@@ -18,7 +18,7 @@
 else
 	libdir="/usr/lib"
 fi
-Libs="-L$libdir @UPSCLI_LIBOBJ@ @SSL_LDFLAGS@"
+Libs="-L$libdir @UPSCLI_LIBOBJ@ @LIBSSL_LDFLAGS@"
 
 
 case "$1" in
--- nut-2.2.0/lib/libupsclient.pc.in.wrongssl	2007-09-06 11:08:09.000000000 +0200
+++ nut-2.2.0/lib/libupsclient.pc.in	2007-09-06 11:08:30.000000000 +0200
@@ -9,5 +9,5 @@
 Name: libupsclient
 Description: UPS monitoring with Network UPS Tools
 Version: @PACKAGE_VERSION@
-Libs: -L${libdir} -lupsclient @SSL_LDFLAGS@
-Cflags: -I${includedir} @SSL_CFLAGS@
+Libs: -L${libdir} -lupsclient @LIBSSL_LDFLAGS@
+Cflags: -I${includedir} @LIBSSL_CFLAGS@
