diff -up nut-2.8.3-build/nut-2.8.3/drivers/Makefile.am.rhinoname nut-2.8.3-build/nut-2.8.3/drivers/Makefile.am
--- nut-2.8.3-build/nut-2.8.3/drivers/Makefile.am.rhinoname	2025-05-21 13:16:02.566975097 +0200
+++ nut-2.8.3-build/nut-2.8.3/drivers/Makefile.am	2025-05-21 13:17:07.563528645 +0200
@@ -62,7 +62,7 @@ NUTSW_DRIVERLIST = $(NUTSW_DRIVERLIST_DU
 SERIAL_DRIVERLIST = al175 bcmxcp belkin belkinunv bestfcom	\
  bestfortress bestuferrups bestups etapro everups 	\
  gamatronic genericups isbmex liebert liebert-esp2 liebert-gxe masterguard metasys	\
- mge-utalk microdowell microsol-apc mge-shut oneac optiups powercom rhino	\
+ mge-utalk microdowell microsol-apc mge-shut oneac optiups powercom nutdrv_rhino	\
  safenet nutdrv_siemens-sitop solis tripplite tripplitesu upscode2 victronups powerpanel \
  blazer_ser ivtscd apcsmart apcsmart-old riello_ser sms_ser bicker_ser
 if HAVE_LINUX_SERIAL_H
@@ -182,8 +182,8 @@ powercom_SOURCES = powercom.c
 powercom_LDADD = $(LDADD) -lm
 powerpanel_SOURCES = powerpanel.c powerp-bin.c powerp-txt.c
 powerpanel_LDADD = $(LDADD) -lm
-rhino_SOURCES = rhino.c
-rhino_LDADD = $(LDADD) -lm
+nutdrv_rhino_SOURCES = rhino.c
+nutdrv_rhino_LDADD = $(LDADD) -lm
 safenet_SOURCES = safenet.c
 nutdrv_siemens_sitop_SOURCES = nutdrv_siemens_sitop.c
 solis_SOURCES = solis.c
