diff -up nut-2.7.1/scripts/systemd/nut-driver.service.in.systemdfix nut-2.7.1/scripts/systemd/nut-driver.service.in
--- nut-2.7.1/scripts/systemd/nut-driver.service.in.systemdfix	2013-07-26 21:41:11.000000000 +0200
+++ nut-2.7.1/scripts/systemd/nut-driver.service.in	2012-07-31 19:38:56.000000000 +0200
@@ -4,7 +4,7 @@ After=local-fs.target network.target
 StopWhenUnneeded=yes
 
 [Service]
-ExecStart=@DRVPATH@/upsdrvctl start
-ExecStop=@DRVPATH@/upsdrvctl stop
+ExecStart=@driverexecdir@/upsdrvctl start
+ExecStop=@driverexecdir@/upsdrvctl stop
 Type=forking
 
