diff -up nut-2.7.3/scripts/systemd/nut-server.service.in.systemdfix nut-2.7.3/scripts/systemd/nut-server.service.in
--- nut-2.7.3/scripts/systemd/nut-server.service.in.systemdfix	2015-04-28 09:59:05.567849447 +0200
+++ nut-2.7.3/scripts/systemd/nut-server.service.in	2015-04-28 09:59:23.743808581 +0200
@@ -4,7 +4,7 @@ After=local-fs.target network.target nut
 # We don't Require drivers to be successfully started! This would be
 # a change of behavior compared to init SysV, and could prevent from
 # accessing successfully started, at least to audit a system.
-#Requires=nut-driver.service
+Wants=nut-driver.service
 Before=nut-monitor.service
 
 [Service]
